jaas/src/main/java/com/griddynamics/jagger/jaas/JaasStarter.java

```diff
@@ -0,0 +1,33 @@
+package com.griddynamics.jagger.jaas;
+
+import com.griddynamics.jagger.dbapi.DatabaseService;
+import com.griddynamics.jagger.engine.e1.services.DataService;
+import com.griddynamics.jagger.engine.e1.services.DefaultDataService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ImportResource;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.annotation.PropertySources;
+
+/**
+ * Spring Boot based starter.
+ */
+@PropertySources({@PropertySource("classpath:/basic/default.environment.properties")})
+@ImportResource({"classpath:/common/storage.rdb.client.conf.xml", "classpath:/spring/dbapi.config.xml"})
```

Comment:
Yes I see, thank you.

I have just noticed that earlier we had a handmade PropertyPlaceholder and decided to make sure, that the functionality is not lost / or is still needed. https://github.com/griddynamics/jagger/blob/master/chassis/configuration/configuration/basic/properties.resolver.conf.xml